src/libs/actions/BankAccounts.js

```diff
@@ -483,6 +500,24 @@ function setFreePlanVerifiedBankAccountID(bankAccountID) {
     API.SetNameValuePair({name: CONST.NVP.FREE_PLAN_BANK_ACCOUNT_ID, value: bankAccountID});
 }
 
+/**
+ * @param {Number} bankAccountID
+ * @param {String} validateCode
+ */
+function validateBankAccount(bankAccountID, validateCode) {
+    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {loading: true});
+
+    API.BankAccount_Validate({bankAccountID, validateCode})
+        .then((response) => {
+            if (response.jsonCode === 200) {
+                Growl.show('Bank Account succesfully validated!', CONST.GROWL.SUCCESS, 3000);
+                Navigation.dismissModal();
```

Comment:
NAB, because I think this works for now. But just leaving a mental note that we might want to show some kind of "success" screen. Since the plan is to enable Expensify cards automatically and there won't be anything to block the user from returning to this flow necessarily and I was thinking it would show a permanent view for the `EnableStep` just saying something like "there's no more actions to take" or whatever.